public/dom.js

```diff
@@ -1,37 +1,41 @@
 const locateMe = document.querySelector('.button__location');
```

Comment:
Please change to the same method of getting the variable. :)

Reply:
hcanged the last one on line 4 to query selector